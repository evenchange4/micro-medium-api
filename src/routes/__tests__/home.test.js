const home = require('../home');

jest.mock('micro', () => ({
  send: (res, code, content) => ({ res, code, content }),
}));

it('should return home', () => {
  expect(home()).toMatchSnapshot();
});
