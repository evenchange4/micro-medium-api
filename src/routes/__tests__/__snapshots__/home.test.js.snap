// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return home 1`] = `
Object {
  "code": 200,
  "content": "Please try /@:username/posts?limit=100",
  "res": undefined,
}
`;
